modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_service_base.rb

```diff
@@ -44,6 +46,33 @@ def get_error_message(error)
         end
       end
 
+      def get_error_key(error_message)
+        error_message.dig(:messages, 0, :key) || error_message
```

Comment:
Not sure if it's really an issue (and if it's within scope of this ticket) - should we return an `errors[]` instead of `[error]`? I.E. if `messages[].length > 1` should we return more than the first one?

Reply:
I believe this always receives just _one_ error_message generated from _one_ issue.  I'm not sure there is any handling for piling up multiple errors, at least not at the moment